actions.jsx

```diff
@@ -189,7 +191,39 @@ export function setIndicator(indicator) {
 }
 
 
+export function fetchApplicationBootstrap() {
+  return dispatch => {
+    dispatch(requestApplicationBootstrap());
+    const bootstrapEndpoint = $.ajax({
+      type: 'GET',
+      url: 'https://nxt.staging.lizard.net/bootstrap/kpi/',
+      xhrFields: {
+        withCredentials: true,
+      },
+      success: (data) => {
+        return data;
+      },
+    });
```

Comment:
Heb ook geprobeerd om `dispatch()` in de $.ajax.success() handler te stoppen maar ook dat geeft problemen.